Source/WebCore/css/CSSFontFaceSrcValue.cpp

```diff
@@ -72,17 +72,18 @@ bool CSSFontFaceSrcLocalValue::equals(const CSSFontFaceSrcLocalValue& other) con
     return m_fontFaceName == other.m_fontFaceName;
 }
 
-CSSFontFaceSrcResourceValue::CSSFontFaceSrcResourceValue(ResolvedURL&& location, String&& format, LoadedFromOpaqueSource source)
+CSSFontFaceSrcResourceValue::CSSFontFaceSrcResourceValue(ResolvedURL&& location, String&& format, Vector<FontTech> technologies, LoadedFromOpaqueSource source)
```

Comment:
I think this creates a copy of the vector, not sure this is desirable.

Reply:
I don't think so, I'm creating a rvalue reference when invoking this constructor so there will be 2 moves but no copy.

Reply:
Tim is right, this causes a copy. The receiver and the sender both have to cooperate to move

Reply:
Why not unify the format and the technologies into the same data structure?